### PR TITLE
Release 1.3.3

### PR DESCRIPTION
**Pull request by SushiRoll53**
* Fixed provider registration.
* Fixed import generation.
* Figma now removed shadows from image.
* Fixed state management bugs.
* Fixed PBDL bugs.

### CODE REVIEW

File: lib/generation/generators/util/pb_generation_view_data.dart [hunk 1]
```diff
@@ -77,11 +77,22 @@ class PBGenerationViewData {
...
+    var result = false;
+    _methodVariables.forEach((element) {
+      if (element.variableName == variable.variableName) {
+        result = true;
+        return result;
+      }
+    });
+    return result;
+  }
+
```
Comment (SuperGNUS) [inline lib/generation/generators/util/pb_generation_view_data.dart]: I would suggest you use a Map or Set, probably won't matter too much for the amount of variables in this case, but it's good practice to reduce the search complexity to O(log n).  The underlying binary search tree for these items uses a red-black tree.

Comment (siliconivan) [reply lib/generation/generators/util/pb_generation_view_data.dart]: Good point. @ivan-015 @SushiRoll53 shall we make an issue for this to be refactored?

Comment (ivan-015) [reply lib/generation/generators/util/pb_generation_view_data.dart]: Sounds good, I'll create an issue for a hotfix
Edit: [here](https://github.com/Parabeac/Parabeac-Core/issues/263) is the issue

**Review by ivan-015 — APPROVED** [review]
LGTM

**Review by siliconivan — APPROVED** [review]
lgtm

**Pull request merged**